### PR TITLE
Small edits to the pipeline description

### PR DESCRIPTION
**Pull request by mbookman**

### CODE REVIEW

File: docs/source/use_cases/analyze_reads/count_reads.rst [hunk 1]
```diff
@@ -25,7 +25,8 @@ Setup Dataflow
 Run the pipeline
 ----------------
 
-The following command will count reads from a BAM in `Google Cloud Storage`_, specifically those in the BRCA1 region for sample NA12877 within the `Platinum Genomes`_ dataset:
+The following command will count reads from a BAM in `Google Cloud Storage`_,
```
Comment (pgrosu) [inline docs/source/use_cases/analyze_reads/count_reads.rst]: It looks fine but usually it's written as: "from a BAM file".  Maybe having the following instead but not required:

The following command will count reads from a BAM file located in `Google Cloud Storage`_,

The only other thing, why the underscore (i.e.  `Google Cloud Storage`_,)?

File: docs/source/use_cases/analyze_reads/count_reads.rst [hunk 2]
```diff
@@ -6,14 +6,14 @@ Count Reads
 
 .. contents::
 
-This simple pipeline just count reads and can be run either against a BAM file
+This simple pipeline counts reads and can be run either against a BAM file
 in `Google Cloud Storage`_ or against data accessed via the `Google Genomics Reads API`_.
```
Comment (pgrosu) [inline docs/source/use_cases/analyze_reads/count_reads.rst]: Looks good but why the underscores?

**Comment by deflaux** [comment]
@mbookman looks good, do you mind retracting this and re-sending to the staging branch?

@pgrosu the underscores are reStructured text markup.  The urls are [here](https://github.com/googlegenomics/start-here/blob/master/docs/source/conf.py#L123)

**Comment by pgrosu** [comment]
I learn something new every day :)  Thanks @deflaux!

**Pull request closed without merging**